### PR TITLE
[OLD] [CC-33996] Adds keyboard accessibility

### PR DESCRIPTION
**Pull request by fedets-spryker**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/CC-33995

###### Optional

- Core PR: https://github.com/spryker/suite-nonsplit/pull/9262

###### Change log

{Relevant changes for project level go here.}

### CODE REVIEW

File: src/Pyz/Yves/ShopUi/Theme/default/components/molecules/slick-carousel/slick-carousel.scss [hunk 1]
```diff
@@ -119,28 +119,29 @@
                     margin-top: rem(24);
                 }
 
+                button {
+                    content: '';
```
Comment (supproduction) [inline src/Pyz/Yves/ShopUi/Theme/default/components/molecules/slick-carousel/slick-carousel.scss:123]: Do you need it?

File: src/Pyz/Yves/ShopUi/Theme/default/components/molecules/slick-carousel/slick-carousel.scss [hunk 2]
```diff
@@ -119,28 +119,29 @@
...
+                    width: rem(5);
+                    height: rem(5);
+                    background: $setting-color-light;
+                    border-radius: 50%;
+                    border: 2px solid $setting-color-transparent;
```
Comment (supproduction) [inline src/Pyz/Yves/ShopUi/Theme/default/components/molecules/slick-carousel/slick-carousel.scss:128]: Please, use rems

File: src/Pyz/Yves/ShopUi/Theme/default/components/molecules/date-time-picker/date-time-picker.ts [hunk 3]
```diff
@@ -1,115 +1,71 @@
...
     protected trigger: HTMLInputElement;
     protected dateFrom: HTMLInputElement;
     protected dateTo: HTMLInputElement;
+    protected datepicker: HTMLInputElement;
 
     protected readyCallback(): void {}
 
     protected init(): void {
-        this.trigger = <HTMLInputElement>this.getElementsByTagName('input')[0];
+        this.trigger = <HTMLInputElement>this.querySelector('input[type="text"]');
```
Comment (supproduction) [inline src/Pyz/Yves/ShopUi/Theme/default/components/molecules/date-time-picker/date-time-picker.ts:12]:
```suggestion
        this.trigger = this.querySelector<HTMLInputElement>('input[type="text"]');
```
Please, fix everywhere

File: src/Pyz/Yves/ProductImageWidget/Theme/default/components/molecules/image-gallery/image-gallery.ts [hunk 4]
```diff
@@ -38,9 +44,16 @@ export default class ImageGallery extends Component {
...
+    protected onThumbnailButtonHover(event: Event): void {
+        const target = event.target as HTMLButtonElement;
+        // target.closest('.slick-slide').dispatchEvent(new Event('mouseenter'));
+        $(target).parent('.slick-slide').trigger('mouseenter');
+    }
+
     protected onThumbnailHover(event: Event): void {
         const slide = $(event.currentTarget);
         const index = slide.data('slick-index');
+        console.log($(event.currentTarget));
```
Comment (supproduction) [inline src/Pyz/Yves/ProductImageWidget/Theme/default/components/molecules/image-gallery/image-gallery.ts:56]: Please, clean it up

File: src/Pyz/Yves/ProductImageWidget/Theme/default/components/molecules/image-gallery/image-gallery.ts [hunk 5]
```diff
@@ -38,9 +44,16 @@ export default class ImageGallery extends Component {
         this.setDefaultImageUrl();
     }
 
+    protected onThumbnailButtonHover(event: Event): void {
+        const target = event.target as HTMLButtonElement;
+        // target.closest('.slick-slide').dispatchEvent(new Event('mouseenter'));
+        $(target).parent('.slick-slide').trigger('mouseenter');
```
Comment (supproduction) [inline src/Pyz/Yves/ProductImageWidget/Theme/default/components/molecules/image-gallery/image-gallery.ts:50]: Can we use native js instead?

**Review by supproduction — CHANGES_REQUESTED** [review]

File: src/Pyz/Yves/CatalogPage/Theme/default/templates/page-layout-catalog/page-layout-catalog.twig [hunk 6]
```diff
@@ -3,6 +3,12 @@
 {% block contentClass %}page-layout-main page-layout-main--catalog-page{% endblock %}
 
 {% block content %}
+    {% include atom('skip-link', '@SprykerShop:ShopUi') with {
```
Comment (tolerants) [inline src/Pyz/Yves/CatalogPage/Theme/default/templates/page-layout-catalog/page-layout-catalog.twig:6]:
```suggestion
    {% include atom('skip-link') with {
```

File: src/Pyz/Yves/CatalogPage/Theme/default/templates/page-layout-catalog/page-layout-catalog.twig [hunk 7]
```diff
@@ -71,7 +77,7 @@
                 {% endblock %}
             </div>
 
-            <div class="col col--sm-12 col--lg-8 col--xl-9">
+            <div class="col col--sm-12 col--lg-8 col--xl-9" id="products-catalog">
```
Comment (tolerants) [inline src/Pyz/Yves/CatalogPage/Theme/default/templates/page-layout-catalog/page-layout-catalog.twig:80]:
```suggestion
            <div class="col col--sm-12 col--lg-8 col--xl-9" id="catalog-product-list">
```

File: src/Pyz/Yves/CompanyPage/Theme/default/templates/page-layout-company/page-layout-company.twig [hunk 8]
```diff
@@ -43,6 +43,12 @@
         {% block inner %}
             <div class="grid grid--gap">
                 <div class="col col--sm-12 col--xl-3 is-hidden-sm-lg">
+                    {% include atom('skip-link', '@SprykerShop:ShopUi') with {
+                        data: {
+                            href: '#company-content',
+                            text: 'global.skip-to-company-content.link.text' | trans,
```
Comment (tolerants) [inline src/Pyz/Yves/CompanyPage/Theme/default/templates/page-layout-company/page-layout-company.twig:49]: Do we need a specific text for it?

File: src/Pyz/Yves/ProductImageWidget/Theme/default/components/molecules/image-gallery/image-gallery.ts [hunk 9]
```diff
@@ -38,9 +44,16 @@ export default class ImageGallery extends Component {
         this.setDefaultImageUrl();
     }
 
+    protected onThumbnailButtonHover(event: Event): void {
+        const target = event.target as HTMLButtonElement;
+        // target.closest('.slick-slide').dispatchEvent(new Event('mouseenter'));
```
Comment (tolerants) [inline src/Pyz/Yves/ProductImageWidget/Theme/default/components/molecules/image-gallery/image-gallery.ts:49]: Forgot to remove?

File: src/Pyz/Yves/ShopUi/Theme/default/components/molecules/date-time-picker/date-time-picker.ts [hunk 10]
```diff
@@ -1,115 +1,71 @@
 import Component from 'ShopUi/models/component';
-import $ from 'jquery/dist/jquery';
-import 'jquery-datetimepicker/build/jquery.datetimepicker.full';
 
 export default class DateTimePicker extends Component {
     protected trigger: HTMLInputElement;
     protected dateFrom: HTMLInputElement;
     protected dateTo: HTMLInputElement;
+    protected datepicker: HTMLInputElement;
```
Comment (tolerants) [inline src/Pyz/Yves/ShopUi/Theme/default/components/molecules/date-time-picker/date-time-picker.ts:7]:
```suggestion
    protected datePicker: HTMLInputElement;
```

File: src/Pyz/Yves/ShopUi/Theme/default/components/molecules/date-time-picker/date-time-picker.ts [hunk 11]
```diff
@@ -1,115 +1,71 @@
...
 
     protected init(): void {
-        this.trigger = <HTMLInputElement>this.getElementsByTagName('input')[0];
+        this.trigger = <HTMLInputElement>this.querySelector('input[type="text"]');
         this.dateFrom = <HTMLInputElement>document.getElementById(this.dateFromId);
         this.dateTo = <HTMLInputElement>document.getElementById(this.dateToId);
+        this.datepicker = <HTMLInputElement>this.querySelector(`.${this.jsName}__datepicker`);
+        this.datepicker.value = this.trigger.value;
 
+        this.datetimepickerInit();
```
Comment (tolerants) [inline src/Pyz/Yves/ShopUi/Theme/default/components/molecules/date-time-picker/date-time-picker.ts:18]: Please choose datePicker or dateTimePicker.

File: src/Pyz/Yves/ShopUi/Theme/default/components/molecules/slick-carousel/slick-carousel.scss [hunk 12]
```diff
@@ -119,28 +119,29 @@
...
+
+                    &[aria-selected="true"] {
```
Comment (tolerants) [inline src/Pyz/Yves/ShopUi/Theme/default/components/molecules/slick-carousel/slick-carousel.scss:130]: Would it work?
```suggestion
                    &[aria-selected] {
```

**Review by tolerants — CHANGES_REQUESTED** [review]

File: src/Pyz/Yves/CompanyPage/Theme/default/templates/page-layout-company/page-layout-company.twig (hunk 8 above)
Comment (fedets-spryker) [reply src/Pyz/Yves/CompanyPage/Theme/default/templates/page-layout-company/page-layout-company.twig:49]: Replaced with more general text version to use on both - company and customer pages

**Pull request closed without merging**